specification/netapp/resource-manager/readme.md

```diff
@@ -83,7 +93,11 @@ These settings apply only when `--tag=package-preview-2023-05` is specified on t
 ``` yaml $(tag) == 'package-preview-2023-05'
 input-file:
   - Microsoft.NetApp/preview/2023-05-01-preview/netapp.json
+<<<<<<< HEAD
+tag: package-2024-03
+=======
 tag: package-preview-2024-03
+>>>>>>> b57df44178d5249021b520845425671bb7507dd7
```

Comment:
please fix these merge validation errors. These are preventing the swagger lint check from running

Reply:
Thats a strange conflict,  fixed